# id of generating task
int8 task_id

action_dispatcher/SceneSetup start_scene

# Trajectory display stuff
action_dispatcher/MinimalSubTrajectory[] sub_trajectories